changelog/4620.feature.rst

```diff
@@ -0,0 +1,5 @@
+sunpy now sets auxillary parameters using the `astropy.wcs.wcs.aux` attribute.
+This stores observer information, along with the reference solar radius if present.
+
+This means the sunpy specific attributes ``.heliographic_observer`` and ``.rsun``
+are no longer set on the `~astropy.wcs.WCS` returned by `sunpy.map.Map.wcs`.
```

Comment:
duplication with the other entry? Do we need this here?

Reply:
This is a feature entry, and the other one is a behaviour change entry. I think we should have both (so the changes show up in both sections), but happy to wordsmith either message.

Reply:
I get having both entries, I am just not sure they need the same info in both. We could have implemented this feature while still keeping these attributes, so not saying this in the feature entry seems ok to me?